pkg/model/crd.go

```diff
@@ -359,6 +359,54 @@ func (r *CRD) ExceptionCode(httpStatusCode int) string {
 	return "UNKNOWN"
 }
 
+// Return true if all the required status fields are missing from ReadOneInput. Else, return false
```

Comment:
top tip for Golang...

Godoc expects docstring comments for all targets (methods, functions, types and fields) to be in the form `{{ Target }} blah blah`. So, for this method, the docstring should be like this:

```go
// GoCodeRequiresStatusFieldsMissingFromReadOneInput returns true if all the
// required status fields are missing...
```

This expected format helps godoc generate documentation for various targets and keeps the docstrings consistent.

But, remember that this function doesn't actually return true or false :) It returns a **string** with Go code that returns true or false ;) So, that docstring isn't quite true (pun intended).